common/build-style/python3-module.sh

```diff
@@ -17,6 +17,20 @@ do_build() {
 	fi
 }
 
+do_check() {
+	local _default_target=test
+
+	if [ -z "$make_check_target" ]; then
+		if ! python3 setup.py --help ${_default_target} >/dev/null 2>&1; then
+			msg_warn "No command '${_default_target}' defined by setup.py.\n"
+			return 0
+		fi
+	fi
+
+	: ${make_check_target:=${_default_target}}
```

Comment:
put it in place of _default_target and use it, set test as default_target.

users in the template just need to change make_check_target